### ISSUE TITLE
rename 'Submit bug report' to 'Submit debug logs'

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#5985]
...and give a link to github for submitting actual bug reports instead, in order to simplify our triage process

**Comment by uhoreg** [comment]
IMHO, it's nice to have an in-app way to submit bug reports, as some people refuse to use GitHub, or some don't know how to.  And for some, creating yet-another-account may be a sufficient barrier to stop them from submitting a bug report at all.

**Comment by lampholder** [comment]
We've wrestled long and hard with this, and have concluded (for now at least) that the signal-to-noise ratio on bug reports via the 'Submit bug report' button is bad enough that it's not serving its purpose as a useful stream of feedback. So we've decided to encourage people to go via github (where there's a bug-filing template, etc.) to improve the general quality of bugs (at the cost of exluding those users who aren't willing to use github).

We should certainly continue to review this decision as the product evolves and its user base continues to expand.
